### PR TITLE
Fix overlapping issue on mobile

### PR DESCRIPTION
**Pull request by rafaucau**
- Fixed the best answer block that overlaps the post meta on mobile

<details>
<summary>Before</summary>

</details>

<details>
<summary>After</summary>

</details>

### CODE REVIEW

File: resources/less/forum.less [hunk 1]
```diff
@@ -60,10 +60,16 @@
   }
 }
 
-@media only screen and (max-device-width: 480px) {
+@media only screen and (max-device-width: 560px) {
```
Comment (askvortsov1) [inline resources/less/forum.less:63]: Could we use the variables from core? I think those should be available.

Comment (rafaucau) [reply resources/less/forum.less:63]: It looks best on 580px width, but if other extensions add their items there, there may be a problem with overlapping again.
So it's a good idea with these core variables. I will rebuild these styles a bit.

Comment (rafaucau) [reply resources/less/forum.less]: I added some changes. Now the div with the best answer goes to the second line only if it doesn't fit in the first line

**Comment by KyrneDev** [comment]
I'm not a huge fan of the badge getting pushed to an entirely new line. Space is limited on mobile and I think it takes up too much on it. 

How about you just hide the time or who marked it as the best answer or both (depending on space) on mobile.

**Comment by rafaucau** [comment]
@KyrneDev Good idea. But on the smallest screens it will go to the new line anyway.
In addition, the translation "best answer" is quite long in some languages.

So I'll hide the time and who marked it on mobile, but it won't solve the problem completely.

**Comment by rafaucau** [comment]
Now it looks like this:

**Comment by KyrneDev** [comment]
Yup, that's money, thanks!

**Pull request merged**